scripts/apps/ingest/services/SendService.ts

```diff
@@ -212,10 +212,13 @@ export function SendService(
                     $rootScope.$broadcast('item:update', {item: _item});
                     return _item;
                 });
-        } else if (!item.lock_user) {
+        } else {
             return api.save('move', {}, {task: data, allPackageItems: config.sendAllPackageItems}, item)
                 .then((_item) => {
                     $rootScope.$broadcast('item:update', {item: _item});
+                    if (item.lock_user) {
```

Comment:
you need to also check if the same item is open in authoring. Otherwise it would close authoring with article A open when you send article B to another stage when it is locked, but not open in authoring.